app/models/user.rb

```diff
@@ -70,7 +74,19 @@ def password_reset_expired?
   end
 
   def feed
-    microposts
+    Micropost.where("user_id IN (?) OR user_id = ?", following_ids, id)
```

Comment:
where("user_id IN (?) OR user_id = ?", following_ids, id) => please use scope to define it

Reply:
and use scope here